### PR TITLE
Add packing list for burn night dinner

### PR DESCRIPTION
**Pull request by sarina**
@ywwg it's important that you don't forget this

i'm new to go so feel free to give me comments to fix things around!

### CODE REVIEW

**Comment by sarina** [comment]
I thought of this when I woke up - I need advice on how to handle the "rate" parameter on the structs, because these are consumables but actually kind of not consumed at a rate, they'll (ideally) all go at once.

**Comment by ywwg** [comment]
Then you want a CustomConsumableItem!  I did this for gasoline.  It lets you pass an inline calculation function for your rate calculation, taking the number of days as a parameter.

File: src/github.com/ywwg/items/friday_dinner.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	plib "github.com/ywwg/packinglib"
+)
+
+var burnDinner = []plib.Item{
+	plib.NewConsumableItem("pulled pork", 8.5, "pounds", []string{"Burn"}, nil),
+	plib.NewConsumableItem("vegan chili", 3.5, "gallons", []string{"Burn"}, nil),
+	plib.NewConsumableItem("pulled jackfruit", 50, "ounces", []string{"Burn"}, nil),
+	plib.NewConsumableItem("shredded cheese", 2, "pounds", []string{"Burn"}, nil),
+	plib.NewConsumableItem("sour cream", 24, "ounces", []string{"Burn"}, nil),
+	plib.NewBasicItem("sarina's cooler", []string{"Camping"}, nil),
```
Comment (ywwg) [inline src/github.com/ywwg/items/friday_dinner.go]: Oh cool so now whenever I go camping I will always bring your cooler :p

j/k, I know I need to figure out things like "one-time" packable items.  There's also the question of how to handle forks of this repo, but let everyone have their own packing lists.

**Comment by ywwg** [comment]
Oh haha I probably shouldn't have merged, it's telling me to bring "34 pounds of pulled pork".  "The tests will come in a later PR"

**Comment by sarina** [comment]
lol I can make a new PR with revised changes, no worries. I am sitting next to a former Googler who is excited to get me set up with a Go env

**Comment by ywwg** [comment]
sorry I fixed it already but Patches Welcome if you come up with something better than the gross thing I did

**Comment by ywwg** [comment]
(a new type for one-time consumables seems reasonable)

**Pull request merged**